core/sys/cc.h

```diff
@@ -129,12 +129,16 @@
 
 #define CC_CONCAT2(s1, s2) s1##s2
 /**
- * A C preprocessing macro for concatenating to
- * strings.
+ * A C preprocessing macro for concatenating two strings.
  *
  * We need use two macros (CC_CONCAT and CC_CONCAT2) in order to allow
  * concatenation of two \#defined macros.
  */
 #define CC_CONCAT(s1, s2) CC_CONCAT2(s1, s2)
```

Comment:
The name of this macro is not well chosen since the same kind could be added for the 3-token version. This macro just makes it possible to use `CC_CONCAT2()` from anywhere, even outside the context of another macro having `s1` and `s2` as parameters.

Maybe it could be renamed to something like `CC_CONCAT_EXT_2`.